OpenXMLSDK.Engine/Word/ReportEngine/Renders/PageExtensions.cs

```diff
@@ -42,6 +42,28 @@ public static void Render(this Page page, Models.Document document, OpenXmlEleme
                 };
                 sectionProps.AppendChild(pageMargins);
             }
+
+            // If Columns are defined on the page we split page in columns
+            if (context.TryGetItem(page.ColumnNumberKey, out DoubleModel ColumnNumberKey))
+            {
+                // Try to convert double to ColumnCountValues : 1, 2 or 3
```

Comment:
You can merge this if statement with the one above

Reply:
Ok, changed.